OracleWebLogic/samples/wls-k8s-domain/README.md

```diff
@@ -0,0 +1,137 @@
+WebLogic Sample on Kubernetes with Shared Domain Home
+=========================================
+This sample extends the Oracle WebLogic developer install image by creating a sample WLS 12.2.1.3 domain and cluster to run in Kubernetes. The WebLogic domain consists of an Admininstrator Server and several Managed Servers running in a WebLogic cluster. All WebLogic servers share the same domain home which has been mapped to an external volume.
+
+## Prerequisites
+1. You need to have a Kubernetes cluster up and running with kubectl installed.
+2. You have built oracle/weblogic:12.2.1.3-developer image locally based on Dockerfile and scripts here: https://github.com/oracle/docker-images/tree/master/OracleWebLogic/dockerfiles/12.2.1.3/
+3. Username/password for the WebLogic domain are stored in k8s/secrets.yml and they are encoded by base64. The default values are weblogic/weblogic1.  
+If you want to customize it, first get the encoded data of your username/password via running `echo -n <username> | base64` and `echo -n <password> | base64`. Next upate k8s/secrets.yml with the new encoded data.
+
+## How to Build and Run
+
+### 1. Build the WebLogic Image for This Sample Domain
+Pre-steps before build the image:
+1. You need to download get-pip.py from https://bootstrap.pypa.io/get-pip.py and save it to folder 'container-scripts'.
+2. If you run `docker build` behind a proxy, you need to set up http and https proxy in the Dockerfile.
+
+Then build the image:
+```
+$ docker build -t wls-k8s-domain .
+```
+Or you can build the image by running build.sh directly.
+
+### 2. Prepare Volume Directories
+Three volumes are defined in k8s/pv.yml which refer to three external directories. You can choose to use host paths or shared NFS directories. Please change the paths accordingly. The external directories need to be initially empty.
+
+**NOTE:** The first two persistent volumes 'pv1' and 'pv2' will be used by WebLogic server pods. All processes in WebLogic server pods are running with UID 1000 and GID 1000 by default, so proper permissions need to be set to these two external directories to make sure that UID 1000 or GID 1000 have permission to read and write the volume directories. The third persistent volume 'pv3' is reserved for later use. We assume that root user will be used to access this volume so no particular permission need to be set to the directory.  
+ 
+### 3. Deploy All the Kubernetes Resources
+Run the script deploy.sh to deploy all resources to your Kubernetes cluster. You can also deploy the resources indivisually by running the following commands:
```

Comment:
The `deploy.sh` script just contains the exact commands as below, so there is no point in having it. If you provide a script, it should do validation and error checking as well.

Also, spelling error: should be _individually_.